CHANGELOG.adoc

```diff
@@ -15,16 +15,19 @@ toc::[]
 == https://github.com/d4l-data4life/hc-fhir-helper-sdk-kmp/compare/v1.6.0...master[Unreleased]
 
 === Added
+
 * link:https://github.com/d4l-data4life/hc-result-sdk-kmp/[Data4Life Result-SDK] (Dependency).
 
 === Changed
+
 * _**BREAKING**_ DataRestrictionException is now a domain error of this SDK (`care.data4life.sdk.helpers.lang.DataRestrictionException`)
 
 === Removed
 
 === Fixed
 
 === Bumped
+
```

Comment:
there is no entry for the bumped dependency

Reply:
Indeed, it got just added with the #23.